`make opt` fails after ce29581f2c

```
$ make world
...
$ make opt
...
../boot/ocamlrun ../ocamlopt -nostdlib -I ../stdlib -for-pack Ocamlbuild_pack -warn-error A -w L -w R -w Z -I ../otherlibs/unix -safe-string -c ocamlbuild.ml
../boot/ocamlrun ../ocamlopt -nostdlib -I ../stdlib -I ../otherlibs/unix -o ocamlbuild.native \
          unix.cmxa ocamlbuild_pack.cmx ocamlbuild_plugin.cmx ocamlbuild_executor.cmx ocamlbuild_unix_plugin.cmx ocamlbuild.cmx
../stdlib/libasmrun.a(minor_gc.o): In function `promote_stack':
/home/samir/src/ocaml-multicore/asmrun/minor_gc.c:109: undefined reference to `caml_clean_stack_domain'
collect2: error: ld returned 1 exit status
...
```

--- a/byterun/fiber.c
+++ b/byterun/fiber.c
@@ -37,6 +37,11 @@ void caml_clean_stack(value stack)
 {
   caml_fatal_error("Fibers unimplemented");
 }
+
+void caml_clean_stack_domain(value stack, struct domain* domain)
+{
+  caml_fatal_error("Fibers unimplemented");
+}
 #else
 
 /* One word at the base of the stack is used to store the stack pointer */

